1-Split-Array/Array split.py

```diff
@@ -0,0 +1,34 @@
+def splitArray(array, arrayType):
+	positive = []
+	negative = []
+	try :
+		for element in array:
+			element = int(element)
+			if element >= 0:
+				positive.append(element)
+			elif element < 0:
+				negative.append(element)
+		if arrayType == "+":
```

Comment:
remove that if, you should return 2 arrays in class instead calling function twice
```
class numbers: 
 def __init__(self): 
  self.positive = []
  self.negative = []

def split(): 
... set self.positive and self.negative here
 return numbers()
```